Explorer/View/TableViewCell.swift

```diff
@@ -28,6 +29,12 @@ class TableViewCell: UITableViewCell {
         diveLogView.timestampLabel.text = viewModel.date
         diveLogView.usernameLabel.text = viewModel.username
         
+        if let mapUrl = viewModel.mapImageUrl {
+            diveLogView.bgImageView.af_setImage(withURL: mapUrl)
+        } else {
+            diveLogView.bgImageView.image = #imageLiteral(resourceName: "ScreenshotGoogleMaps")
```

Comment:
You should probably use a different default image - something that makes it clear there's no map image available.